### PR TITLE
git-svn: add page

### PR DESCRIPTION
**Pull request by joelhy**
Add git-svn command to common page.

### CODE REVIEW

**Comment by joelhy** [comment]
origin from https://github.com/tldr-pages/tldr/pull/486

File: pages/common/git-svn.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# git svn
+
+> Bidirectional operation between a Subversion repository and Git
+
+- clone a SVN repository
```
Comment (waldyrious) [inline pages/common/git-svn.md]: "a**n** SVN repository"

File: pages/common/git-svn.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+`git svn clone {{http://example.com/my_subversion_repo}} {{local_dir}}`
+
+- update from SVN repository
```
Comment (waldyrious) [inline pages/common/git-svn.md]: maybe "update local clone from the upstream SVN repository"

File: pages/common/git-svn.md (hunk 1 above)
Comment (joelhy) [inline pages/common/git-svn.md]: Fixed.

File: pages/common/git-svn.md (hunk 2 above)
Comment (joelhy) [inline pages/common/git-svn.md]: Wording changed, thanks.

**Comment by rubenvereecken** [comment]
Thanks, looks good :+1:. I was just about to common on "an SVN" but I see I was looking at an outdated diff.

**Comment by felixonmars** [comment]
:+1:

**Pull request merged**